src/components/pages/tests.js

```diff
@@ -72,10 +72,10 @@ function Tests({ PageTemplate, initialState }) {
     window.alert('Started DocumentRequests creation, check your console!');
   };
 
-  // StarRating
-  const [starRatingValue, setStarRatingValue] = useState(3.7);
-  const [starRatingCompact, setStarRatingCompact] = useState(false);
-  const [starRatingTotalCount, setStarRatingTotalCount] = useState(15);
+  // DocumentRating
+  const [documentRatingValue, setDocumentRatingValue] = useState(3.7);
```

Comment:
I think now we could remove the fake initial values (3.7 for rating and the 15 for the total count), right?

Reply:
As it is just the UI component without any document connection, I just left it in as example values, as the component looks nicer when initially displayed that way :-) But of course we could set it to `null` and `0` if you'd prefer.

Reply:
🤦‍♀️ Test page, I missed that part, sorry!